cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/service/naming/GcpInstanceGroupResourceName.java

```diff
@@ -0,0 +1,57 @@
+package com.sequenceiq.cloudbreak.cloud.gcp.service.naming;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GcpInstanceGroupResourceName {
+    private static final Logger LOGGER = LoggerFactory.getLogger(GcpInstanceGroupResourceName.class);
+
+    private static final int INSTANCE_GROUP_PART_COUNT = 3;
+
+    private final String stackName;
+
+    private final String groupName;
+
+    private final String suffix;
+
+    public GcpInstanceGroupResourceName(String stackName, String groupName, String suffix) {
+        this.stackName = stackName;
+        this.groupName = groupName;
+        this.suffix = suffix;
+    }
+
+    public static GcpInstanceGroupResourceName instanceGroupResourceNameFromString(String cloudResourceName) {
```

Comment:
POJO class should not contain logic

Reply:
removed. I don't take it as POJO class though.